libs/ngx-mime/src/lib/core/intl/help-intl.ts

```diff
@@ -10,4 +10,5 @@ export class HelpIntl {
   line8 = '<strong>P</strong>: Previous result';
   line9 = '<strong>F</strong>: Fullscreen on/off (Close with <strong>F</strong> or <strong>ESC</strong>.)';
   line10 = '<strong>R</strong>: Rotate 90°';
+  line11 = '<strong>T</strong>:  Show/hide optically recognized text (for content that can be displayed)';
```

Comment:
To get screen readers to pause after an sentence, we should have a period punctuation at the end of each sentence.
Parenthesis are not read by the screen readers. 
PS: I have only tested with a free [Chrome extension](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn)

```suggestion
  line11 = '<strong>T</strong>:  Show/hide optically recognized text (for content that can be displayed).';
```